packages/toolkit/.vscodeignore

```diff
@@ -2,6 +2,10 @@
 *
 */**
 
+# Ignore from the root package.json otherwise those dirs will be collected as well.
+../../*
+../../*/**
```

Comment:
would this be fixed by invoking `vsce` from this directory as the CWD? I wonder if that would avoid surprises in general.

Reply:
> Its not entirely clear to me why vsce includes parent directories, but researching has led me to answers that [vsce does not officially support workspaces](https://github.com/microsoft/vscode-vsce/issues/777).

Nice. Would be useful to include that comment + URL in the code comment.